libs/sdk-ui/api/sdk-ui.api.md

```diff
@@ -45,16 +45,16 @@ import { IPositiveAttributeFilter } from '@gooddata/sdk-model';
 import { IPreparedExecution } from '@gooddata/sdk-backend-spi';
 import { IRankingFilter } from '@gooddata/sdk-model';
 import { IRelativeDateFilter } from '@gooddata/sdk-model';
-import { IResultAttributeHeader } from '@gooddata/sdk-backend-spi';
-import { IResultHeader } from '@gooddata/sdk-backend-spi';
-import { IResultMeasureHeader } from '@gooddata/sdk-backend-spi';
-import { IResultTotalHeader } from '@gooddata/sdk-backend-spi';
-import { IResultWarning } from '@gooddata/sdk-backend-spi';
+import { IResultAttributeHeader } from '@gooddata/sdk-model';
+import { IResultHeader } from '@gooddata/sdk-model';
+import { IResultMeasureHeader } from '@gooddata/sdk-model';
+import { IResultTotalHeader } from '@gooddata/sdk-model';
+import { IResultWarning } from '@gooddata/sdk-model';
 import { ISeparators } from '@gooddata/sdk-backend-spi';
```

Comment:
Please check the ISeparators, they should now only come from sdk-model.

Reply:
Removed.